src/splitInteger.test.js

```diff
@@ -4,18 +4,27 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(20, 5);
 
+  result.forEach((value) => expect(result[0]).toBe(value));
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const value = 8;
+  const result = splitInteger(value, 1);
 
+  expect(result[0]).toBe(value);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const result = splitInteger(33, 5);
 
+  expect(result).toEqual([6, 6, 7, 7, 7]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
+  const result = splitInteger(3, 4);
 
+  expect(result[0]).toBe(0);
 });
```

Comment:
The test description 'should add zeros if value < numberOfParts' is slightly misleading. The test checks only the first element for zero, but it would be more comprehensive to check the entire array to ensure all parts are correct. Consider updating the expectation to check the entire result array, e.g., `expect(result).toEqual([0, 1, 1, 1]);`.